src/server.js

```diff
@@ -132,61 +128,47 @@ export default class Tapestry {
         match({
           routes: DefaultRoutes(this.config.components || {}),
           location: request.url.path
-        }, (error, redirectLocation, renderProps) => {
+        }, (err, redirectLocation, renderProps) => {
+
+          // define global deets for nested components
+          const loadContext = this.config
+
+          // 404 if non-matched route
+          if (!renderProps) {
+            return reply(
+              renderHtml({
+                loadContext,
+                assets: this.env === 'production' ? this.assets : null
+              })
+            ).code(404)
+          }
 
           // 500 if error from Router
-          if (error)
-            return reply(error.message).code(500)
+          if (err) {
+            error(err)
```

Comment:
Will the `process.exit(0)` in these error calls kill the server? Will the return statement run?